lib/Checks.groovy

```diff
@@ -1,9 +1,40 @@
+import org.yaml.snakeyaml.Yaml
+
 /*
  * This file holds several functions used to perform standard checks for the nf-core pipeline template.
  */
 
 class Checks {
 
+    static void check_conda_channels(log) {
+        Yaml parser = new Yaml()
+        def channels = []
+        try {
+            def config = parser.load("conda config --show channels".execute().text)
+            channels = config.channels
+        } catch(NullPointerException | IOException e) {
+            log.warn("Could not verify conda channel configuration.")
```

Comment:
```suggestion
            log.warn "Could not verify conda channel configuration."
```

Reply:
Think that should work, and makes things consistent with other instances of using `log.warn` in the script.